lib/rules/wrap-iife.js

```diff
@@ -46,64 +56,96 @@ module.exports = {
                 nextToken && nextToken.value === ")";
         }
 
-        return {
+        /**
+        * Get the function node from an IIFE
+        * @param {ASTNode} node to evaluate
+        * @returns {ASTNode} node that is the function expresion of the given IIFE, or null if non exist
+        */
+        function getFunctionNodeFromIIFE(node) {
```

Comment:
This is a direct rip from https://github.com/jscs-dev/node-jscs/blob/master/lib/utils.js#L79